packages/player-example/src/App.tsx

```diff
@@ -440,8 +449,8 @@ const ControlsOnly: React.FC<{
 				.reverse()
 				.slice(0, 10)
 				.reverse()
-				.map((l) => {
-					return <div key={l}>{l}</div>;
+				.map((l, i) => {
+					return <div key={`${l}-${i}`}>{l}</div>;
```

Comment:
Scale change when resizing up were alright but resizing down meant that some log content was the exact same (since scales were repeated)... this should fix that 😊

Reply:
Very nice! 💡 I noticed it was buggy!